src/sota_tools/authenticate_test.cc

```diff
@@ -42,12 +44,12 @@ TEST(authenticate, good_cert_zip) {
 
   EXPECT_EQ(CURLE_OK, rc);
 }
-*/
 
 /* Authenticate with nothing (no auth).
  * Parse authentication information from treehub.json.
  * Parse images repository URL from a provided archive. */
 TEST(authenticate, good_cert_noauth_zip) {
+  // Authenticates with ssl_noauth_server on port 2443.
```

Comment:
Just a minor nitpicking: would be better to use "tls" instead of "ssl" everywhere to be consistent with kTls and such.

Reply:
Fixed.